hack/build-jsonnet.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 set -e
-set -x
+# set -x
```

Comment:
Why are we disabling it?

Reply:
oh, good catch, i disabled it because it was super chatty for me, i can reenable it.